include/api.php

```diff
@@ -5490,6 +5490,36 @@ function api_friendica_profile_show($type)
 }
 api_register_func('api/friendica/profile/show', 'api_friendica_profile_show', true, API_METHOD_GET);
 
+/**
+ * Returns a list of saved searches.
+ *
+ * @see https://developer.twitter.com/en/docs/accounts-and-users/manage-account-settings/api-reference/get-saved_searches-list
+ *
+ * @param  string $type Return format: json or xml
+ *
+ * @return string|array
+ * @throws UnauthorizedException
```

Comment:
Where does it throw this exception?

Reply:
Not directly in this function but since `$auth` is set to `true` in `api_register_func()`, it will throw an exception if you try to call this API without being logged in.

Reply:
Then the hint should be on whatever function actually throws the exception, shouldn’t it?

Reply:
I thought the point of `@throws` was to list potential exceptions even if they are not in the function's code so you could easily see what exceptions could occur when calling a function.
But I looked at the phpdoc reference and I feel like you're right.

I will have to check my other PRs for wrong `@throws` statement as well.